src/components/notifications/AppExplorer/AppCard/AppCard.scss

```diff
@@ -44,6 +44,7 @@
       color: var(--fg-color-2);
     }
     &__subscribe {
+      margin-top: 0.67em;
```

Comment:
Why do we need this

Reply:
<img width="316" alt="image" src="https://github.com/WalletConnect/web3inbox/assets/26746725/4e66ef96-f583-4635-b798-33353cd91b94">
its cuz of the spacing between the button and text